docker-build

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+# The docker repo.
+REPO="$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME"
+
+# Usage prints the help for this command.
+usage() {
+  echo "Usage:"
+  echo "    docker-build command"
+  echo ""
+  echo "Commands:"
+  echo "    build  Build a docker image from this repo and tag it with the git sha and branch"
+  echo "    push   Push the built image to the docker registry"
```

Comment:
Maybe put a comment in here about all the variables that are necessary for this to run?